shared/teams/new-team/wizard/new-team-info.tsx

```diff
@@ -151,9 +194,13 @@ const NewTeamInfo = () => {
           checked={openTeam}
           onCheck={v => (rolePickerIsOpen ? undefined : setOpenTeam(v))}
         />
+        {teamWizardState.teamType === 'subteam' && (
+          <Kb.Checkbox onCheck={setAddYourself} checked={addYourself} label="Add yourself to the team" />
```

Comment:
as per discussion could be nice to add `LayoutAnimation` to the open team dropdown. with this checkbox added the extra empty space is apparent.